internal/widget/reddit.go

```diff
@@ -17,6 +18,7 @@ type Reddit struct {
 	Style         string          `yaml:"style"`
 	Limit         int             `yaml:"limit"`
 	CollapseAfter int             `yaml:"collapse-after"`
+	HTTPProxy     string          `yaml:"proxy"`
```

Comment:
Small nitpick, but we might add other types of proxies later on. 

```suggestion
	HTTPProxy     string          `yaml:"http-proxy"`
```

Reply:
I think in that case, it's probably a good idea to call it `custom-endpoint` rather than `http-proxy`, allowing the user to change the whole endpoint.